### ISSUE TITLE
Disabling geonode.monitoring causes model exceptions on views

### ISSUE THREAD

**Issue by afabiani** [GeoNode/geonode#5096]
## Actual Behavior
When running GeoNode with default settings (sqlite and moniroting disabled), db operations rise exceptions on monitoring receivers

![image](https://user-images.githubusercontent.com/1278021/67287566-a41eae00-f4db-11e9-92ae-fb9766cbca34.png)

## Steps to Reproduce the Problem

  1. run geonode vanilla, default settings
  2. create a new user
  3. try to delete the new created user from the admin dsahboard

## Specifications

  - GeoNode version: 2.10.1
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker): any
  - Platform: any
  - Additional details:
Basically the model expects the monitoring tables to be there, because the views now rely on "monitoring.registered_event" to perform logging.
There are controls inside the methods in order to skip operations if some of the monitoring apps are disabled, but still, the monitoring model is required.
